app/main.py

```diff
@@ -1,8 +1,49 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
```

Comment:
It is a conventional way to place all arguments including self on a new line in the `__init__` so that there would be no arguments on the same lines with opening and closing parenthesis. Rewrite `__init__` here and in other classes to encounter that.

Reply:
@Dimosphen1 +